temporalcli/commandsgen/commands.yml

```
@@ -374,6 +375,99 @@ commands:
     option-sets:
       - workflow reference
 
+  - name: temporal activity pause
+    summary: Pause an Activity
+    description: |
+      Pausing an activity means:
+        - If the activity is currently waiting for a retry or is running 
+      and subsequently fails, it will not be rescheduled until it is unpaused.
+      - If the activity is paused, calling this method will have no effect.
+      - If the activity is running and finishes successfully, 
+        the activity will be completed.
+      - If the activity is running and finishes with failure:
+        * if there is no retry left - the activity will be completed.
+        * if there are more retries left - the activity will be paused.
+      
+      Specify the
+      Activity and Workflow IDs:
+
+      ```
+      temporal activity pause \
+          --activity-id YourActivityId \
+          --workflow-id YourWorkflowId
+      ```
+    options:
+      - name: activity-id
+        short: a
+        type: string
+        description: Activity ID to pause.
+        required: true
+      - name: identity
+        type: string
+        description: Identity of the user submitting this request.
+    option-sets:
+      - workflow reference
+
+  - name: temporal activity unpause
+    summary: Unpause an Activity
+    description: |
+      Unpauses the execution of an activity specified by its ID.
+      Returns a `NotFound` error if there is no pending activity 
+      with the provided ID.
+      There are two 'modes' of unpausing an activity:
+        'resume' - Default. If the activity is paused, it will be resumed 
+        and scheduled for execution.
+        * If the activity is currently running Unpause with 'resume' has 
+          no effect.
+        * if 'no_wait' flag is set and the activity is waiting, the activity 
+          will be scheduled immediately.
+        'reset' - If the activity is paused, it will be reset to its initial 
+        state and (depending on parameters) scheduled for execution.
+        * If the activity is currently running, Unpause with 'reset' will reset 
+          the number of attempts.
+        * if 'no_wait' flag is set, the activity will be scheduled immediately.
+        * if 'reset_heartbeats' flag is set, the activity heartbeat timer 
+          and heartbeats will be reset.
+      If the activity is in waiting for retry and past it retry timeout, 
+      it will be scheduled immediately.
+      Once the activity is unpaused, all timeout timers will be regenerated.
+
+      Specify the
+      Activity and Workflow IDs:
```

Same as above: this is going to be quite hard to read, and we also want to up-level docs to focus more on the effects/outcomes a command has, rather than what it does or how it works.

```suggestion
      Re-schedule a previously-paused Activity for execution.

      If the Activity is not running and is past its retry timeout, it will be
      scheduled immediately. Otherwise, it will be scheduled after its retry
      timeout expires. The Activity can be retried immediately with `--no-wait`.

      Use `--reset` to reset the number of previous run attempts to zero. For
      example, if an Activity is near the maximum number of attempts N specified
      in its retry policy, `--reset` will allow the Activity to be retried
      another N times after unpausing.

      Specify the Activity and Workflow IDs:
```